src/session/content/chat_history.rs

```diff
@@ -67,6 +74,13 @@ mod imp {
                         Chat::static_type(),
                         glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
                     ),
+                    glib::ParamSpecBoolean::new(
+                        "sticky",
+                        "Sticky",
+                        "Whether the room history should stick to the newest message in the timeline",
+                        true,
+                        glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
```

Comment:
```suggestion
                        glib::ParamFlags::READABLE,
```

I think that this property should only be updated internally. Also, the setter below should be private.